Backend-Development/Job-Boards/Readme.md

```diff
@@ -4,4 +4,5 @@
 - [Backend Developer](https://www.glassdoor.co.in/Job/backend-developer-jobs-SRCH_KO0,17.htm) - Glassdoor.com
 - [Backend Engineer](https://angel.co/role/backend-engineer) - Angel.co
 - [Backend Developer](https://www.turing.com/jobs/remote-back-end-developer-jobs) - Turing.com
-- [LinkedIn Job Board](https://www.linkedin.com/jobs/search/?geoId=92000000&keywords=backend%20developer&location=Worldwide)
\ No newline at end of file
+- [LinkedIn Job Board](https://www.linkedin.com/jobs/search/?geoId=92000000&keywords=backend%20developer&location=Worldwide)
+- [GFG Job Board](https://practice.geeksforgeeks.org/jobs?ref=ghm)
```

Comment:
Remove `?ref=ghm` from the link. Also, the portal shows mixed results and not only for the backend. Could you please add filters so it only shows backend jobs when visited on the link? you can customize filters and add it to different job boards respectively

Reply:
The page is filtering within itself without making changes in the actual link, I don't think we can filter the search through the URL.